std/bigint.d

```diff
@@ -716,8 +716,11 @@ public:
             throw new FormatException("Format specifier not understood: %" ~ f.spec);
 
         char[] buff =
-            hex ? data.toHexString(0, '_', 0, f.flZero ? '0' : ' ')
-                : data.toDecimalString(0);
+            f.spec == 'X' ?
+                data.toHexString(0, '_', 0, f.flZero ? '0' : ' ', LetterCase.upper) :
+            f.spec == 'x' ?
+                data.toHexString(0, '_', 0, f.flZero ? '0' : ' ', LetterCase.lower) :
+                data.toDecimalString(0);
```

Comment:
Is it really necessary to expand the original code into this hard-to-read cascade of ternary `?` operators, instead of rewriting it with if-statements?